src/devices/vdml_motorgroup.cpp

```diff
@@ -561,7 +562,7 @@ std::int32_t MotorGroup::set_gearing(const motor_gearset_e_t gearset, const std:
 }
 std::int32_t MotorGroup::set_gearing(std::vector<motor_gearset_e_t> gearsets) const {
 	empty_MotorGroup_check(PROS_ERR);
-	for (int i = 0; i < gearsets.size(); i++) {
+	for (uint32_t i = 0; i < gearsets.size(); i++) {
```

Comment:
```suggestion
	for (size_t i = 0; i < gearsets.size(); i++) {
```
This is a bit pedantic but `size_t` is the proper type for indices; if you're going to change it, might as well make it the right type.

Reply:
`std::size_t` for consistent style - looks like we're using `std::` prefixed integer types in this file (as per the function signature).

Should add `#include <cstddef>` in that case instead of `<cstdint>`, though seems like we should have `<cstdint>` too anyway..